### ISSUE TITLE
Filter by Ongoing Requests in Live View

### ISSUE THREAD

**Issue by tamir-alltrue-ai** [pydantic/logfire#445]
### Description

It would be very useful to filter to ongoing requests only in the live view. I often times want to see the logs corresponding with actions that I'm taking in my client - especially long-lived operations - and being able to filter to live requests would help me identify the spans corresponding with the actions I'm taking in real time.

**Comment by alexmojaki** [comment]
~Try `kind = 'pending_span'`.~

The above does NOT work, even though it looks like it does.

cc @petyosi who I think would be interested in the existence of this use case.

**Comment by tamir-alltrue-ai** [comment]
Excellent, thanks!

The next thing that would be great would be to filter to "top level" spans  - when I run that query I see all sub-spans unfolded by default, but what I would love to do is 

`kind = 'pending_span' and span_level = 0` or something

**Comment by alexmojaki** [comment]
Right-click on your profile pic and turn off "Expand to display query-matching spans", then refresh the page.

The way to identify those spans in SQL is `parent_span_id is null`, but for technical reasons your experience will probably be smoother if you don't also include that in your filter in the live view.

**Comment by dmontagu** [comment]
Closing — answered above: `kind = 'pending_span'`, and `parent_span_id is null` for top-level spans.
